src/Modules/Payments/Application/Subscriptions/RenewSubscription/SubscriptionRenewedEnqueueEmailConfirmationHandler.cs

```diff
@@ -26,32 +27,14 @@ public class SubscriptionRenewedEnqueueEmailConfirmationHandler : INotificationH
         
         public async Task Handle(SubscriptionRenewedNotification notification, CancellationToken cancellationToken)
         {
-            var payer = await GetPayerDetails(notification.DomainEvent.PayerId);
+            PayerEmailProvider.Init(_sqlConnectionFactory);
+            
+            var payer = await PayerEmailProvider.GetPayerDetails(notification.DomainEvent.PayerId);
```

Comment:
This is not a good solution, it introduces temporal coupling https://blog.ploeh.dk/2011/05/24/DesignSmellTemporalCoupling/

Just pass `_sqlConnectionFactory` as an argument to `PayerEmailProvider.GetPayerDetails` method :)

Reply:
And the method should return string and name should be GetPayerEmail

Reply:
I think it's done ;)